core/src/poh_service.rs

```diff
@@ -89,15 +89,18 @@ impl PohService {
         let poh = poh_recorder.lock().unwrap().poh.clone();
         let mut now = Instant::now();
         let mut num_ticks = 0;
+        let mut num_hashes = 0;
         loop {
+            num_hashes += NUM_HASHES_PER_BATCH;
```

Comment:
does this need to get reset when `num_ticks` does?

Reply:
yes :)